web/docs/source/GetStartedGuide.rst

```diff
@@ -0,0 +1,138 @@
+Getting Started
+===============
+
+Setup Chronos for development
+-----------------------------
+
+The first step when setting up Chronos is to clone the repository:
+
+.. code-block:: bash
+    
+    git clone https://github.com/teknik-eksjo/chronos.git
+
+Create a virtual python environment, activate it and install the python project requirements:
+
+.. code-block:: bash
+    
+    cd chronos/web
+    python3 -m venv venv
+    . venv/bin/activate
+    pip install -r requirements.txt
+
+If needed, also install docker-compose:
+
+.. code-block:: bash
+
+    pip install docker-compose
```

Comment:
https://docs.docker.com/compose/install/#alternative-install-options doesn't elaborate that much but recommends against a system-wide install with pip.

https://docs.docker.com/compose/completion/ might also be of interest.